docs/concepts/07_dip/05_dapp_developer.md

```diff
@@ -26,8 +26,128 @@ Import the SDK into your code:
 import { * } from '@kiltprotocol/dip-sdk'
 ```
 
+## Example application
+
+This example application walks through the code you can find in the tests for the DIP SDK.
+
+### 1. Generate a base proof
+
+Start by generating the base proof for the DIP using the `[generateDipSiblingBaseProof](https://kiltprotocol.github.io/dip-sdk/functions/generateDipSiblingBaseProof.html)` method and passing the desired configuration:
+
+```typescript
+const baseDipProof = await DipSdk.generateDipSiblingBaseProof(config)
+```
+
+:::info What's a base proof?
+A base proof is a cross-chain state proof, revealing the parts of a DID stored on the KILT blockchain.
+:::
+
+The configuration for the base proof takes the following parameters:
+
+-   `DipSiblingBaseProofInput`: The input for the base proof. It can be one of the following values:
```

Comment:
Maybe we can remove this level of nesting?

Reply:
@ntn-x2  What do you mean? The bullets? This is the formatting from markdown lint and conforms to our various style rules.

Reply:
Yes but do we need the top level `DipSiblingBaseProofInput`? That's just the name given to the config object, which is actually whatever is nested under this bullet point. Unless you want to maintain the naming as exposed by the DIP-SDK, in that case it would make sense to leave it as-is.